configurations/otelcol/windows/default/apica/otelcol/config.yaml

```diff
@@ -275,8 +239,8 @@ receivers:
             metric: system.processor.queue.length
           - name: "Context Switches/sec"
             metric: system.context.switches.sec
-          # - name: "System Calls/sec"
```

Comment:
Keep an eye on this one when we turn it on. I've seen errors come up in the otel collector logs about this metric having a negative value. Not saying not to turn it on, just something to keep an eye on.